thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/tools/FetchAutoTuneResult.java

```diff
@@ -82,7 +91,40 @@ public static void main(String[] args) throws Exception{
 
     // write to file
     String header = "FunctionId" + "," + AutoTuneAlertFilterTool.EvaluationNode.getCSVSchema() + "," + "isModelUpdated" + "," +AutoTuneAlertFilterTool.EvaluationNode.getCSVSchema();
-    executor.writeMapToCSV(outputMap, DIR_TO_FILE + Collection + CSVSUFFIX, header);
-    LOG.info("Write evaluations to file: {}", DIR_TO_FILE + Collection + CSVSUFFIX);
+    executor.writeMapToCSV(outputMap, fileName, header);
+    LOG.info("Write evaluations to file: {}", fileName);
   }
+
+
```

Comment:
please provide necessary comments and description such that others can leverage on this tool you wrote.